packages/pwa-kit-react-sdk/package.json

```diff
@@ -7,6 +7,11 @@
     "node": "^14.0.0",
     "npm": "^6.14.4 || ^7.0.0 || ^8.0.0"
   },
+  "typedoc": {
+    "entryPoint": "./src/index.ts",
+    "readmeFile": "./README.md",
+    "displayName": "PWA Kit React SDK"
+  },
```

Comment:
Newb question: do we need this here at the moment? We're only setting up typedoc for the commerce-sdk-react package for now, no?

Reply:
Good catch, we don't need this here for now. I will remove it